src/renderers/common/CubeRenderTarget.js

```diff
@@ -72,6 +78,91 @@ class CubeRenderTarget extends WebGLCubeRenderTarget {
 
 	}
 
+	fromCubeTexture( renderer, cubeTex, sigmaRadians = 0, poleAxis = new Vector3( 0, 1, 0 ) ) {
+
+		const currentGenerateMipmaps = cubeTex.generateMipmaps;
+
+		cubeTex.generateMipmaps = true;
+
+		this.texture.type = cubeTex.type;
+		this.texture.colorSpace = cubeTex.colorSpace;
+
+		this.texture.generateMipmaps = cubeTex.generateMipmaps;
+		this.texture.minFilter = cubeTex.minFilter;
+		this.texture.magFilter = cubeTex.magFilter;
+
+		// The maximum length of the blur for loop. Smaller sigmas will use fewer
+		// samples and exit early, but not recompile the shader.
+		const MAX_SAMPLES = 20;
+
+		const blurMaterial = new NodeMaterial();
+		blurMaterial.side = BackSide;
+		blurMaterial.depthTest = false;
+		blurMaterial.depthWrite = false;
+		blurMaterial.blending = NoBlending;
+
+		const weights = uniformArray( new Array( MAX_SAMPLES ).fill( 0 ) );
+		const dTheta = uniform( 0 );
+		const n = float( MAX_SAMPLES );
+		const latitudinal = uniform( 0 ); // false, bool
+		const samples = uniform( 1 ); // int
+		const envMap = TSL_CubeTexture( null );
+
+		const cubeSampler = Fn( ( [ sampleDirection ] )=>{
+
+			return envMap.sample( sampleDirection );
+
+		} );
+		blurMaterial.fragmentNode = blur( { n, latitudinal: latitudinal.equal( 1 ), poleAxis: vec3( poleAxis ), outputDirection: positionWorldDirection, weights, samples, dTheta, sampler: cubeSampler } );
+
+		const geometry = new BoxGeometry( 5, 5, 5 );
+		const mesh = new Mesh( geometry, blurMaterial );
+
+		const scene = new Scene();
+		scene.add( mesh );
+
+		const camera = new CubeCamera( 1, 10, this );
+
+		const width = cubeTex.source.data[0].width;
```

Comment:
I've run into this issue before in Three (and complained about it) - this feels super fragile, but how am I supposed to find the size of a cubeTexture? Some have `image`, some have `source`, and I'm pretty sure if you generate one with a render target it has none? Seems like we're missing an API for this. Can we please just have a `.size`?